types/passport-openidconnect/index.d.ts

```diff
@@ -0,0 +1,121 @@
+    );
+
+    authenticate(req: Request, options?: any): any;
+    authorizationParams(options: any): object;
+}
+
+declare namespace OpenIDConnectStrategy {
+    type Profile = passport.Profile;
+
+    type VerifyCallback = (err?: Error | null, user?: Express.User, info?: any) => void;
+
+    type VerifyFunction = (issuer: string, profile: any, done: VerifyCallback) => void;
+
+    type VerifyFunctionWithRequest = (req: Request, issuer: string, profile: any, done: VerifyCallback) => void;
+
+    interface SessionStoreContext {
+        maxAge?: number | undefined;
+        issued?: Date | undefined;
+        nonce?: string;
+    }
+    type SessionStoreStoreCallback = (err: Error | null, handle: string) => void;
+    type SessionStoreVerifyCallback = (err: Error | null, ctx: SessionStoreContext, state: any) => void;
+
+    interface SessionStore {
+        store(
+            req: Request,
+            ctx: SessionStoreContext,
+            appState: any,
+            meta: any,
+            callback: SessionStoreStoreCallback,
+        ): void;
+
+        verify(req: Request, handle: string, callback: SessionStoreVerifyCallback): void;
+    }
+
+    interface _StrategyOptionsBase {
+        issuer: string;
+        authorizationURL: string;
+        tokenURL: string;
+        clientID: string;
+        userInfoURL: string;
+        clientSecret: string;
+        callbackURL: string;
+        /**
+         * `openid` is automatically added.
+         */
+        scope?: string | string[] | undefined;
+        customHeaders?: OutgoingHttpHeaders | undefined;
+        responseMode?: any;
+        prompt?: any;
+        display?: any;
+        uiLocales?: any;
+        loginHint?: any;
+        maxAge?: any;
+        acrValues?: any;
+        idTokenHint?: any;
+        nonce?: any;
+        claims?: any;
+        sessionKey?: string | undefined;
+        store?: SessionStore | undefined;
+        skipUserProfile?: any;
+        proxy?: any;
+        agent?: boolean | Agent | undefined;
+    }
+
+    interface StrategyOptions extends _StrategyOptionsBase {
+        passReqToCallback?: false | undefined;
+    }
+
+    interface StrategyOptionsWithRequest extends _StrategyOptionsBase {
+        passReqToCallback: true;
+    }
+
+    type Strategy = OpenIDConnectStrategy;
+    const Strategy: typeof OpenIDConnectStrategy;
+
+    class AuthorizationError extends Error {
+        constructor(message: string | undefined, code: string, uri?: string, status?: number);
+        code: string;
+        uri?: string | undefined;
+        status: number;
+    }
+
+    class TokenError extends Error {
+        constructor(message: string | undefined, code: string, uri?: string, status?: number);
+        code: string;
+        uri?: string | undefined;
+        status: number;
+    }
+
+    class InternalOAuthError extends Error {
+        constructor(message: string, err: any);
+        oauthError: any;
+    }
```

Comment:
so reall, does it work for you?
```js
const { InternalOAuthError } = require('passport-openidconnect');
```
please do not export symbols that do not exist on native module